models/Task.php

```diff
@@ -966,7 +943,7 @@ public function canResortItems()
      */
     public function getPercent()
     {
-//        $denominator = TaskItem::find()->where(['task_id' => $this->id])->count();
+        //$denominator = TaskItem::find()->where(['task_id' => $this->id])->count();
```

Comment:
remove the comment in line `946`

Reply:
this comment already exists in master branch, it's not me who had done this